src/OctoshiftCLI.Tests/GithubApiTests.cs

```diff
@@ -825,7 +829,7 @@ public async Task DeleteRepo_Calls_The_Right_Endpoint()
             var githubClientMock = new Mock<GithubClient>(null, null, null);
 
             // Act
-            var githubApi = new GithubApi(githubClientMock.Object);
+            var githubApi = new GithubApi(githubClientMock.Object, null);
```

Comment:
shouldn't this be Api_Url instead of null, I'm not sure how this test is even passing like this.

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/af9bce2216bae956787d03c5827047298044ab8b